packages/terra-form/tests/nightwatch/components/fieldset/fieldset-spec.js

```diff
@@ -0,0 +1,29 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
+
+const screenshot = require('terra-toolkit').screenshot;
+
+module.exports = {
+  before: (browser, done) => {
+    browser.resizeWindow(browser.globals.width, browser.globals.height, done);
+  },
+
+  afterEach: (browser, done) => {
+    screenshot(browser, 'terra-form', done);
+  },
+
+  'Displays a default Field': (browser) => {
+    browser
+      .url(`http://localhost:${browser.globals.webpackDevServerPort}/#/tests/form-tests/fieldset/default`)
+      .assert.elementPresent('.terra-Form-field');
+  },
+
+  'Displays a populated Field with correct label, help message, and choices': (browser) => {
+    browser
+      .url(`http://localhost:${browser.globals.webpackDevServerPort}/#/tests/form-tests/fieldset/populated`)
+      .assert.elementPresent('.terra-Form-field')
+      .assert.elementPresent('.terra-Form-field--inline')
+      .assert.containsText('.terra-Form-legend', 'Do you have any Children?')
+      .assert.containsText('.terra-Form-helpText', 'Families are eligible for family package plans')
+      .assert.containsText('.terra-Form-error', 'This field is required');
```

Comment:
![screen shot 2017-05-16 at 1 21 11 pm](https://cloud.githubusercontent.com/assets/2552573/26121717/d0339950-3a3a-11e7-8755-4a13d1b42817.png)

We have fieldset errors looking like this atm.